internal/gen.go

```diff
@@ -56,14 +56,40 @@ func parseQueryReturn(tcf sql_types.TypeConversionFunc, col *plugin.Column) (*co
 		return nil, fmt.Errorf("multidimensional arrays are not supported, store JSON instead")
 	}
 
+	javaType := core.JavaType{
+		SqlType:    sdk.DataType(col.Type),
+		Type:       strJavaType,
+		IsList:     col.IsArray,
+		IsNullable: !col.NotNull,
+	}
+
+	if javaType.IsNullable {
+		if javaType.IsList {
+			nullableHelpers.List = true
+		} else {
+			switch strJavaType {
+			case "Integer":
+				nullableHelpers.Int = true
+				break
+			case "Long":
+				nullableHelpers.Long = true
+				break
+			case "Float":
+				nullableHelpers.Float = true
+				break
+			case "Double":
+				nullableHelpers.Double = true
+				break
+			case "Boolean":
+				nullableHelpers.Boolean = true
+				break
```

Comment:
go doesn't need a `break` statement in the switch case!

Reply:
Removed!